Makefile

```diff
@@ -1,9 +1,23 @@
+detect_long_lines_in_diff:
```

Comment:
Make supports include, so yes, it would be possible to include that in the core bundle, for instance. Or maybe it should go in dev-kit @soullivaneuh?